### PR TITLE
Migrate jellyfinactions

### PR DESCRIPTION
**Pull request by YouKnowBlom**
Migrates jellyfinactions to typescript
Updates all references to jellyfinactions

Refactor to come :)

### CODE REVIEW

File: src/types/global.d.ts [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
+}
+
+export interface GlobalScope {
+    [key: string]: any
+}
+
 declare global {
     export const PRODUCTION: boolean;
     export const RECEIVERVERSION: string;
+    export let $scope: GlobalScope;
```
Comment (ferferga) [inline src/types/global.d.ts:16]:
```suggestion
    export const $scope: GlobalScope;
```

Why not const?

Comment (YouKnowBlom) [reply src/types/global.d.ts:16]: Ah my bad, I thought it was already const. I must've mixed it up with something else :p

**Comment by YouKnowBlom** [comment]
Dang, commiting via the github app just puts a generic commit message. I'll update that later today

**Review by ferferga — APPROVED** [review]

**Pull request merged**